src/test/api/karate/businessconfig/getBusinessconfigHistory.feature

```diff
@@ -0,0 +1,50 @@
+Feature: getBusinessconfig
+
+  Background:
+   #Getting token for admin and operator1_fr user calling getToken.feature
+    * def signIn = callonce read('../common/getToken.feature') { username: 'admin'}
+    * def authToken = signIn.authToken
+    * def signInAsTSO = callonce read('../common/getToken.feature') { username: 'operator1_fr'}
+    * def authTokenAsTSO = signInAsTSO.authToken
+
+
+    Scenario: Push two versions of a bundle 
+          # Push bundle
+      Given url opfabUrl + '/businessconfig/processes'
+      And header Authorization = 'Bearer ' + authToken
+      And multipart file file = {read:'resources/bundle_api_test.tar.gz', contentType: 'application/gzip'}
+      When method post
+      Then status 201
+
+      Given url opfabUrl + '/businessconfig/processes'
+      And header Authorization = 'Bearer ' + authToken
+      And multipart file file = {read:'resources/bundle_api_test_v2.tar.gz', contentType: 'application/gzip'}
+      When method post
+      Then status 201
+
+  Scenario: List existing Businessconfig history
+
+    # Check bundle
+    Given url opfabUrl + '/businessconfig/processes/history/api_test'
+    And header Authorization = 'Bearer ' + authToken
+    When method GET
+    Then status 200
+    And assert response.length == 2
+
+  Scenario: List existing Businessconfig history without authentication
+
+    # Check bundle
+
+    Given url opfabUrl + '/businessconfig/processes/history/api_test'
+    When method GET
+    Then status 401
+
+  Scenario: List not existing Businessconfig history
+
+    # Check bundle
+    Given url opfabUrl + '/businessconfig/processes/history/not_existent'
+    And header Authorization = 'Bearer ' + authToken
+    When method GET
+    Then status 200
```

Comment:
The endpoint businessconfig/processes/{processId} returns 404 in case of process not existing, I think this new endpoint should also return 404 in this case